schema/src/main/jade-tables/clinical_assertion.table.json

```diff
@@ -6,6 +6,11 @@
       "datatype": "string",
       "type": "primary_key"
     },
+    {
+      "name": "release_date",
```

Comment:
Note to self: Need to sweep through these tables and change the partition strategy to use this column

Reply:
Where would that change need to be made?

Reply:
It's in the same file, will comment after I push the changes